CODEOWNERS

```diff
@@ -1 +1 @@
-* @tetratelabs/getistio-team
+* @tetratelabs/getmesh-team
```

Comment:
@deva26 we should have a team wit getistio or whatever the new name 😄

Reply:
let's just remove for now then since this codeowner file is not working as we expected.